### ISSUE TITLE
Error when starting Padrino

### ISSUE THREAD

**Issue by simonc** [nightsailer/padrino-sprockets#3]
Hi, I just installed Padrino (0.10.5) and added padrino-sprockets.
When I start the application I get this weird error:

```
=> Padrino/0.10.5 has taken the stage development at http://0.0.0.0:3000
[2012-01-29 14:53:21] INFO  WEBrick 1.3.1
[2012-01-29 14:53:21] INFO  ruby 1.9.2 (2011-07-09) [x86_64-darwin10.8.0]
[2012-01-29 14:53:21] INFO  WEBrick::HTTPServer#start: pid=76442 port=3000
[2012-01-29 14:53:31] ERROR NoMethodError: undefined method `engines' for Sprockets:Module
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/sprockets-2.3.0/lib/sprockets/environment.rb:39:in `initialize'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/padrino-sprockets-0.0.1/lib/padrino/sprockets.rb:29:in `new'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/padrino-sprockets-0.0.1/lib/padrino/sprockets.rb:29:in `initialize'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:82:in `new'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:82:in `block in use'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:130:in `[]'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:130:in `block in to_app'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:130:in `each'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:130:in `inject'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/builder.rb:130:in `to_app'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/sinatra-1.3.2/lib/sinatra/base.rb:1321:in `new'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/sinatra-1.3.2/lib/sinatra/base.rb:1311:in `prototype'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/sinatra-1.3.2/lib/sinatra/base.rb:1334:in `block in call'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/sinatra-1.3.2/lib/sinatra/base.rb:1416:in `synchronize'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/sinatra-1.3.2/lib/sinatra/base.rb:1334:in `call'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/padrino-core-0.10.5/lib/padrino-core/router.rb:83:in `block in call'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/padrino-core-0.10.5/lib/padrino-core/router.rb:76:in `each'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/padrino-core-0.10.5/lib/padrino-core/router.rb:76:in `call'
    /Users/simonc/.rvm/gems/ruby-1.9.2-p290@blog-2012/gems/rack-1.4.1/lib/rack/handler/webrick.rb:59:in `service'
    /Users/simonc/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/webrick/httpserver.rb:111:in `service'
    /Users/simonc/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/webrick/httpserver.rb:70:in `run'
    /Users/simonc/.rvm/rubies/ruby-1.9.2-p290/lib/ruby/1.9.1/webrick/server.rb:183:in `block in start_thread'
```

Any idea about that ? Thanks

**Comment by nightsailer** [comment]
Do you use compass in your project? 
If so, I think there is problem with it, but not sure the work around,sorry, I'll check it later.

**Comment by simonc** [comment]
Nope, only scss.

**Comment by danpal** [comment]
This is happening on any new padrino, just do:
padrino g project tst -t rspec -s jquery -e haml -c sass  

Add sprockets-padrino afterwards and you'll reproduce

**Comment by danpal** [comment]
Ok this happens on  sprockets 2.3.1.... just add to your gemfile:

```
gem sprockets, "2.3.0" # anchor to 2.3.0
```

then run
    bundle update sprockets

I'll fix this library later to work on 2.3.1. It should also add dependency to sprockets specific version.

**Comment by simonc** [comment]
Yup, working :)
Thanks !
